site2/website/versioned_docs/version-2.4.0/client-libraries-go.md

```diff
@@ -321,6 +337,55 @@ func main() {
 }
 ```
 
+#### Schema example
+
+Here's an example usage of
```

Comment:
it's better to use a whole sentence as the leading-in sentence. such as:

This example shows how to create a producer and consumer with schema.